nixos/modules/misc/ids.nix

```diff
@@ -402,14 +402,14 @@ in
       fourstore = 42;
       fourstorehttp = 43;
       virtuoso = 44;
-      #rtkit = 45; # unused
+      rtkit = 45;
```

Comment:
Why are these being revived?

Reply:
because currently, user `rtkit` has a static uid and group `nogroup`. The static uid is there so that files created by this daemon have a constant numerical owner, so the group `rtkit` I create in this PR must also have a static gid.

It might be the case that neither the user nor the group need a static id at all, but I think this is out of scope of the PR.

Reply:
I'm pretty sure we can let all of these just be allocated dynamically, which we should do according to https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md. You can do that by just not setting a `gid` in the group definition.

Reply:
My point is merely that a static uid is required iff a static gid is required. So I'm not the one to make the decision to remove the static uid. The maintainers of each module should.

Reply:
I don't think a a static uids and gids are linked together? Or where do you see that? Is there an error if you don't assign a static gid?

Reply:
we fix uids because filesystems retain ownership by uid, not user name (as attested by the header of ids.nix, which mentions services with many files). This rationale applies equally to group ownership.

Reply:
does that look reasonable to you, @Infinisil?